credentialproviderpackage/charts/credential-provider-package/templates/daemonset.yaml

```diff
@@ -67,11 +67,6 @@ spec:
         - name: socket
           hostPath:
             path: /run/api.sock
-        {{- else if eq $os "default"}}
```

Comment:
Does Amazon Linux work with "/etc/sysconfig/kubelet"?

Reply:
I'm sorry, I see what you're saying. I'll revert this change.